src/views/navbar/index.js

```diff
@@ -198,19 +246,39 @@ class Navbar extends React.Component<Props> {
     if (!loggedInUser) {
       return (
         <Nav hideOnMobile={hideNavOnMobile} loggedOut={!loggedInUser}>
-          <Logo to="/">
+          <Logo
+            to="/"
+            aria-hidden
+            tabIndex="-1"
+            isHidden={this.state.isSkipLinkFocused}
+          >
             <Icon glyph="logo" size={28} />
           </Logo>
+
+          <SkipLink
+            href="#main"
+            onFocus={this.handleSkipLinkFocus}
+            onBlur={this.handleSkipLinkBlur}
+          >
+            Skip to content
+          </SkipLink>
+
           <HomeTab
             className={'hideOnDesktop'}
             data-active={match.url === '/' && match.isExact}
+            aria-current={
+              match.url === '/' && match.isExact ? 'page' : undefined
```

Comment:
I should probably extract all of those to a method that would return the `data-active` + `aria-current` props, as they use the same condition.